content/sensu-go/6.8/catalog/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Sensu Catalog"
+description: "PLACEHOLDER."
+product: "Sensu Go"
+version: "6.8"
+weight: 55
+layout: "single"
+toc: true
+menu:
+  sensu-go-6.8:
+    identifier: catalog
+---
+
+{{% notice commercial %}}
+**COMMERCIAL FEATURE**: Access the Catalog API in the packaged Sensu Go distribution.
+For more information, read [Get started with commercial features](../../commercial/).
+{{% /notice %}}
+
+{{% notice note %}}
+**NOTE**: The Sensu Catalog is in public preview and is subject to change.
+{{% /notice %}}
+
+The Sensu Catalog...
+
+**PLACEHOLDER**
```

Comment:
If we decide to keep all Catalog docs in a dedicated category, I'll write this page like the rest of the overviews (e.g. https://docs.sensu.io/sensu-go/latest/operations/). For now, it's a placeholder.

Reply:
If we keep the separate Catalog category, I'll also need to create a redirect for https://docs.sensu.io/sensu-go/latest/web-ui/sensu-catalog/ and update links for that page throughout the docs.